src/types/types.ts

```diff
@@ -2,6 +2,15 @@ export type ContextProps = {
   children: React.ReactNode;
 };
 
+export type VoteProps = {
```

Comment:
Rename it to **Vote**.

Reply:
Component name and type name cannot be the same, so I converted them to interface and named them IComponentName.